sdk/react/widgets/VideoViewPlayPause.js

```diff
@@ -144,8 +147,8 @@ class VideoViewPlayPause extends React.Component {
   render() {
     const seekButtonScale = 0.5;
     const playPauseButton = this._renderPlayPauseButton();
-    const backwardButton = this._renderSeekButton(BACKWARD, seekButtonScale);
-    const forwardButton = this._renderSeekButton(FORWARD, seekButtonScale);
+    const backwardButton = this._renderSeekButton(BACKWARD, seekButtonScale, true);
```

Comment:
When forceDvrDisabled is on, both seek buttons should be hidden because you shouldn't be able to scrub back in time. Add the logic for this button too.

Reply:
my bad, realized the logic is correct here. Nothing to do. Sorry @SZadorozhnyiEpam